presto-main/src/main/java/io/prestosql/execution/MemoryTrackingRemoteTaskFactory.java

```diff
@@ -83,10 +83,10 @@ public synchronized void stateChanged(TaskStatus newStatus)
             long currentUserMemory = newStatus.getMemoryReservation().toBytes();
             long currentSystemMemory = newStatus.getSystemMemoryReservation().toBytes();
             long currentRevocableMemory = newStatus.getRevocableMemoryReservation().toBytes();
-            long currentTotalMemory = currentUserMemory + currentSystemMemory;
+            long currentTotalMemory = currentUserMemory + currentSystemMemory + currentRevocableMemory;
```

Comment:
This will need clear explanation in the release notes, so users aren't surprised by the increase (and in 6 months I can remember why it changed)

Reply:
Indeed this needs explanation in notes.
Currently for revocable queries "total" memory usage is sometimes `0` or close to `0` which is even more confusing.
Generally, UI needs a slight re-haul and should not really show `user` `userCumulative` memory anymore. Currently it's not obvious which memory metrics are what in UI. If we make UI to just report total mem that would be much more obvious.